livre/eq/Window.cpp

```diff
@@ -69,10 +69,12 @@ struct Window::Impl
                         pipe->getFrameData()->getVRParameters().getMaxGPUCacheMemoryMB();
         _textureCache.reset( new TextureCache( node->getTextureDataCache(),
                                                maxGpuMemory * LB_1MB, GL_LUMINANCE8 ));
-
-        const size_t computeThreads = 4;
+        const size_t renderThreads = 2;
+        const size_t computeThreads = 2;
         const size_t uploadThreads = 4;
```

Comment:
I prefer functions which are not bloated with parameters, moreover for something that would be nice to use in the future rather than a requirement.